servers/rendering/renderer_viewport.cpp

```diff
@@ -637,6 +637,9 @@ void RendererViewport::viewport_initialize(RID p_rid) {
 	viewport->render_target = RSG::storage->render_target_create();
 	viewport->shadow_atlas = RSG::scene->shadow_atlas_create();
 	viewport->viewport_render_direct_to_screen = false;
+
+	bool is_mobile_renderer = (bool)GLOBAL_GET("rendering/vulkan/rendering/back_end");
```

Comment:
I thik I already commented on this. Again, no clue if this is a reliable and future proof.

Reply:
Changed `is_mobile` check to use `RSG::rasterizer->is_low_end()`